src/Holding/CreateHolding/CreateHolding.test.js

```diff
@@ -0,0 +1,80 @@
+import '../../../test/jest/__mock__';
```

Comment:
Could you please clarify do we need this import? We already define mock in  https://github.com/folio-org/ui-inventory/blob/master/test/jest/jest.setup.js#L1